app/main.ipynb

```diff
@@ -216,15 +277,28 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
-    "# write your code here"
+    "standard_deviations = np.std(A, axis=1)\n",
+    "print(standard_deviations)"
```

Comment:
The calculation of the standard deviation for each row using `np.std` with `axis=1` is correct and matches the task requirement.